flocker/node/agents/ebs.py

```diff
@@ -20,12 +20,15 @@
 from boto.utils import get_instance_metadata
 from boto.exception import EC2ResponseError
 from twisted.python.filepath import FilePath
-from eliot import Field, MessageType
 
 from .blockdevice import (
     IBlockDeviceAPI, BlockDeviceVolume, UnknownVolume, AlreadyAttachedVolume,
     UnattachedVolume, get_blockdevice_volume
 )
+from ._logging import (
+    AWS_ACTION, BOTO_EC2RESPONSE_ERROR, NO_AVAILABLE_DEVICE,
+    NO_NEW_DEVICE_IN_OS
+)
```

Comment:
Tried first attempt at OpenStack ActionType in this branch. The straightforward log change caused a lot of buildbot test failures, which made me realize that adding OpenStack logging will require some code refactoring. Since this branch is getting big, decided to revert OpenStack changes from the branch and do it in a follow up. Please let me know if this does not sound reasonable. Thanks!